librz/core/cmd_descs/cmd_heap_glibc.yaml

```diff
@@ -0,0 +1,86 @@
+---
+name: cmd_heap_glibc
+commands:
+  - name: dmh
```

Comment:
it would be great if you could use `type: RZ_CMD_DESC_TYPE_ARGV_STATE`. You can see `rz_analysis_function_blocks_list_handler`(and others) and read a bit the doc in `rz_cmd.h` on how that changes things.